Backend/mentii/class_ctrl.py

```diff
@@ -0,0 +1,34 @@
+import boto3
+from boto3.dynamodb.conditions import Key, Attr
+from utils.ResponseCreation import ControllerResponse
+from utils import db_utils as dbUtils
+import uuid
+from flask import g
+
+def getActiveClassList(dynamoDBInstance, email=None):
```

Comment:
Both. Its default is none if not provided. It's there for testing purposes only.